README.md

```diff
@@ -117,19 +139,23 @@ from https://github.com/facebook/watchman
 ## Requirements
 
 * [Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
-* `java` and `jar` on the PATH (make sure you use `java` executable from JDK but not JRE)
+* `java` and `jar` on the PATH (make sure you use `java` executable from JDK but not JRE). 
+  * To test this, try running the command `javac`. This command won't exist if you only have the JRE
+  installed. If you're met with a list of command-line options, you're referencing the JDK properly.
+  * If you're running Java 11, you won't be able to build Selenium. We recommend using a tool like  
```

Comment:
This isn't true any more. Hurrah!

Reply:
True, I just tested it. I have Java 8, 11 and 12 installed. I had Java 8 as default, changed it to Java 11, ran `bazel clean` and then `bazel build grid`, and it worked as expected. I can see in the `.bazelrc` file the configuration for the Java version.
@mmerrell therefore this should be removed.